### PR TITLE
Single conf gb training

### PR DESCRIPTION
**Pull request by proteneer**
@chuantian 

An example for training GB parameters on a single conformer

### CODE REVIEW

**Comment by proteneer** [comment]
This needs to be cleaned up extensively but I wanted to get a basic working system out there first.

File: fe/parameterize_simple.py [hunk 1]
```diff
@@ -84,6 +84,23 @@ def close(self):
         PDBFile.writeFooter(self.topology, self.outfile)
         self.outfile.flush()
 
+import jax.numpy as jnp
+
+def loss(du_dls, T, schedule, loss):
```
Comment (chuantian) [inline fe/parameterize_simple.py]: is the input loss true_dG?

File: fe/parameterize_simple.py [hunk 2]
```diff
@@ -152,65 +175,148 @@ def close(self):
...
-                writer = PDBWriter(combined_pdb_str, out_file)
-                # set this to None if we don't care about visualization
-                all_args.append((x0, writer,  conf_idx % num_gpus, precision))
-
-            results = pool.map(sim.run_forward_multi, all_args)
-            all_du_dls.append(results)
+
+    lr = 5e-4
+    # opt_init, opt_update, get_params = optimizers.adam(lr)
+    opt_init, opt_update, get_params = optimizers.sgd(lr)
```
Comment (chuantian) [inline fe/parameterize_simple.py]: I guess we will switch to adam later right?

File: fe/parameterize_simple.py [hunk 3]
```diff
@@ -152,65 +175,148 @@ def close(self):
...
+    host_conf = []
+    for x,y,z in host_pdb.positions:
+        host_conf.append([to_md_units(x),to_md_units(y),to_md_units(z)])
+    host_conf = np.array(host_conf)
+
+    # recenter the ligand based on whether we're using the host or the solvent
+    host_name = "complex"
+    conf_center = conf_com
+
+    init_combined_conf = np.concatenate([host_conf, init_conf])
```
Comment (chuantian) [inline fe/parameterize_simple.py]: This is never used

File: fe/parameterize_simple.py (hunk 1 above)
Comment (proteneer) [reply fe/parameterize_simple.py]: yes sorry - typo

File: fe/parameterize_simple.py (hunk 2 above)
Comment (proteneer) [reply fe/parameterize_simple.py]: for batched, yes

File: fe/parameterize_simple.py (hunk 3 above)
Comment (proteneer) [reply fe/parameterize_simple.py]: ya will clean this up later

**Comment by chuantian** [comment]
are we merging this?

**Comment by proteneer** [comment]
not yet. I need to make some more changes. But feel free to work off the branch

**Comment by proteneer** [comment]
Closing this in favor of #153
